### ISSUE TITLE
Add the possibility to export data

### ISSUE THREAD

**Issue by dshil** [gavv/tagberry#28]
Hi @gavv!

I am looking for a very simple feature. I want to be able to use tagberry on multiple machines by syncing the database files. Currently there is no option/configuration withing which I can setup the folder where to put storage files.

**Comment by gavv** [comment]
> Currently there is no option/configuration withing which I can setup the folder where to put storage files.

Hi! Actually it has one: https://github.com/tagberry/tagberry-qt/blob/main/src/main.cpp#L104

(run --help for details)

**Comment by gavv** [comment]
Note that updating DB on fly wont work. You'll have to shutdown app, update db, and then start the app again.

**Comment by gavv** [comment]
> Add the possibility to export data

Or do you mean explicit export & import functionality?

**Comment by dshil** [comment]
> Or do you mean explicit export & import functionality?

Not exactly. My workflow is the following. Work for some time in tagberry, add some items to the calendar. After that I close the app and run the utility which will push the updated DB files to my google drive account.

Since I installed the tagberry system-wide I do want to have the possibility to change where the DB files are located. I think it is worth to add some kind of settings menu in which we can e.g. configure fonts, DB path, dark/light/system theme, etc.
